embedded-graphics/src/fonts/font6x6.rs

```diff
@@ -0,0 +1,282 @@
+use crate::{fonts::Font, geometry::Size};
+
+/// 6x6 pixel variable width font.
+///
+/// [![6x6 font spritemap screenshot](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/embedded-graphics/data/font6x6.png)](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/embedded-graphics/data/font6x6.png)
```

Comment:
There's a table in the `src/fonts/mod.rs` module docs that needs `Font6x6` adding to it too.

Reply:
Oh, I missed this one indeed. Fixed.